libs/geoservices/covid/src/lib/pages/maps/components/time-map/time-map.component.ts

```diff
@@ -0,0 +1,276 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { Moment } from 'moment';
+import { forkJoin, BehaviorSubject } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+import { MapboxMapService } from '@tamu-gisc/maps/mapbox';
+import { View } from 'typeorm/schema-builder/view/View';
+
+interface StateRecord {
+  STATE: string;
+  pop: number;
+  cases: number;
+  deaths: number;
+  infectionRate: number;
+  deathRate: number;
+}
+
+interface CountyRecord {
+  fips: string;
+  pop: number;
+  cases: number;
+  deaths: number;
+  infectionRate: number;
+  deathRate: number;
+}
+
+interface IInfoBox {
+  name: string;
+  pop: number;
+  cases: number;
+  deaths: number;
+  infectionRate: number;
+  deathRate: number;
+}
+
+@Component({
+  selector: 'tamu-gisc-time-map',
+  templateUrl: './time-map.component.html',
+  styleUrls: ['./time-map.component.scss']
+})
+export class TimeMapComponent implements OnInit {
+  constructor(private mapService: MapboxMapService, private http: HttpClient) {}
+
+  private maxDate: string;
+  private selectedDate: string;
+  private stateData: Array<StateRecord>;
+  private countyData: Array<CountyRecord>;
+
+  @ViewChild('infobox', { static: false }) public infobox: ElementRef;
```

Comment:
Since you're using `infoBoxModel` to render values in the template, this element reference might not be necessary.

Reply:
noted. removed the element reference